src/arrays/Arrays.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace arrays;
+
+class Arrays implements ArraysInterface
+{
+    /**
+     * @param array{name:int,type:int[]} $input
```

Comment:
Please fix annotations for the `$input`, and specify the data type for the return